deployment/hasura/metadata/actions.graphql

```diff
@@ -148,13 +149,18 @@ type SchedulingResponse {
   reason: SchedulingFailureReason
 }
 
-type SchedulingDslTypescriptResponse {
-  status: SchedulingDslTypescriptResponseStatus!
+type DslTypescriptResponse {
+  status: DslTypescriptResponseStatus!
   typescript: String
```

Comment:
Since there's no UI code calling this action yet, let's go ahead and skip the `typescript` field, and just return `typescriptFiles`.

Reply:
That's the same struct returned by the scheduling code action, so it would break the UI code for scheduling DSL.